<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeNewGameDialog</class>
 <widget class="QDialog" name="MakeNewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание новой игры</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QInputDialog QPushButton {
	border-image: url(:/assets/Button.png) 3 10 3 10;
	border-top: 3px transparent;
    border-bottom: 3px transparent;
    border-right: 10px transparent;
    border-left: 10px transparent;
	padding-bottom: 2px;
	font: 9pt &quot;Segoe UI&quot;;
}

QInputDialog QPushButton:pressed {
	border-image: url(:/assets/PressedButton.png)  3 10 3 10;
	border-top: 3px transparent;
    border-bottom: 3px transparent;
    border-right: 10px transparent;
    border-left: 10px transparent;
	padding-bottom: 2px;
	font: 9pt &quot;Segoe UI&quot;;

}

QInputDialog QLabel {
	color: white;
}

QInputDialog QLineEdit {
	background-color: rgb(33, 33, 33);
}


QInputDialog {
	background-color: rgb(33, 33, 33);
}

QDialog {
	background-color: rgb(33, 33, 33);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="whatIsYourNameLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Как вас зовут?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameInput">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(68, 68, 68);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="title">
          <string>Против кого играем?</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="rbVSHuman">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Против человека</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/man.ico</normaloff>:/assets/man.ico</iconset>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbVSCPU">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Против компьютера</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/assets/cpu.ico</normaloff>:/assets/cpu.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="title">
          <string>За кого играем?</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="sideButtonRandom">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Случайно</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sideButtonWhite">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Белые</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sideButtonBlack">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Чёрные</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Выберите уровень сложности</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
background-color: rgb(81, 81, 81);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Придурок</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Легко</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нормально</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Сложно</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Безумно</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Экстремально</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Ознакомьтесь с противником</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QGraphicsView" name="rivalPic">
             <property name="minimumSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>128</width>
               <height>128</height>
              </size>
             </property>
             <property name="styleSheet">
              <string>font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(68, 68, 68);</string>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="rivalName">
             <property name="styleSheet">
              <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Никто не выбран</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="cpuRivalDesc">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(68, 68, 68);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-image: url(:/assets/Button.png) 3 10 3 10;
	border-top: 3px transparent;
    border-bottom: 3px transparent;
    border-right: 10px transparent;
    border-left: 10px transparent;
	padding-bottom: 2px;
	font: 9pt &quot;Segoe UI&quot;;
}

QPushButton:pressed {
	border-image: url(:/assets/PressedButton.png)  3 10 3 10;
	border-top: 3px transparent;
    border-bottom: 3px transparent;
    border-right: 10px transparent;
    border-left: 10px transparent;
	padding-bottom: 2px;
	font: 9pt &quot;Segoe UI&quot;;
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MakeNewGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MakeNewGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
